docs-ref-conceptual/includes/cli-install-linux-script.md

```diff
@@ -11,21 +11,24 @@ ms.custom: devx-track-azurecli, linux-related-content
 
 It's strongly recommend to install the CLI with a package manager. A package manager makes sure you always get the latest updates, and guarantees the stability of CLI components. Check and see if there's a package for your distribution before installing manually.
 
-The CLI requires the following software:
+The install script for the Azure CLI requires the following software:
 
-* [Python 3.6.x, 3.7.x or 3.8.x](https://www.python.org/downloads/).
+* [Python 3.8.x, 3.9.x, 3.10.x](https://www.python.org/downloads/) - For more information, see [Support lifecycle - Python dependency](../azure-cli-support-lifecycle.md#python-dependency).
 * [libffi](https://sourceware.org/libffi/)
 * [OpenSSL 1.0.2](https://www.openssl.org/source/)
 
 ## Install or update Azure CLI
 
-Both installing and updating the Azure CLI requires rerunning the install script.
+> [!IMPORTANT]
+> The install script only works on Python 3.8.x, 3.9.x, or 3.10.x. This install script does not work on Python 3.11.x or later versions.
```

Comment:
The install script works from 3.6 to 3.10.

Reply:
Thanks for the clarification, @bebound. We are removing all references to CentOS b/c of EOL, so I don't think we should mention EOL Python versions here for the same reason. Does this seem logical to you, or what customer value do you see for mentioning 3.6 and 3.7?

Reply:
This is an old script, and I guess most of the customer are using them with old Python version. Removing 3.6 and 3.7 causes confusion.

I'm okay if they are removed.